### ISSUE TITLE
Verify TNEF expansion function

### ISSUE THREAD

**Issue by shawniverson** [E-F-A/v4#255]
https://forum.efa-project.org/viewtopic.php?f=19&p=17266

**Comment by shawniverson** [comment]
group is postfix instead of mtagroup

```
[root@smtp 4DNsyQ0tVTz7g2sY]# ll
total 100
-rw-rw----. 1 postfix postfix     13 Jan 24 07:39 nmsg-362456-1.txt
-rw-rw----. 1 postfix postfix  98184 Jan 24 07:39 nwinmail.dat
drwxrwx--x. 2 postfix mtagroup    80 Jan 24 07:39 tnefTIYQeI
[root@smtp 4DNsyQ0tVTz7g2sY]# cd tnefTIYQeI/
[root@smtp tnefTIYQeI]# ll
total 88
-rw-rw----. 1 postfix postfix 85805 Jan 24 07:39 bookmark.htm
-rw-rw----. 1 postfix postfix  2937 Jan 24 07:39 zappa_av1.jpg
```

**Comment by shawniverson** [comment]
Fixed upstream, prepare to pull into eFa

**Comment by shawniverson** [comment]
Fixed in commit a4b58efc1ed4acb16afed66fe432d0767d281469
